src/components/IndexTable/components/Row/Row.tsx

```diff
@@ -7,16 +7,19 @@ import {
   useIndexSelectionChange,
 } from '../../../../utilities/index-provider';
 import {Checkbox} from '../Checkbox';
-import {classNames} from '../../../../utilities/css';
+import {classNames, variationName} from '../../../../utilities/css';
 import {RowContext, RowHoveredContext} from '../../../../utilities/index-table';
 import styles from '../../IndexTable.scss';
 
+type RowStatus = 'success' | 'disabled';
```

Comment:
Small nitpick - should the status be subdued rather than disabled? I see disabled as a state, while subdued is a variation. What do you think @alex-page ?

```suggestion
type RowStatus = 'success' | 'subdued';
```

Reply:
Yeah this should probably be `subdued`. If you are blocking an interaction you could use `disabled`, however this doesn't seem to be the case. Thanks @AndrewMusgrave